examples/apps/data-object-grid/package.json

```diff
@@ -95,6 +95,7 @@
 		"sass-loader": "^7.1.0",
 		"source-map-loader": "^2.0.0",
 		"style-loader": "^1.0.0",
+		"tinylicious": "^4.0.0",
```

Comment:
Let's remove the start:server script instead, to be consistent with the other demos and the README.md instructions.